bin/templates/project/__PROJECT_NAME__/config.xml

```diff
@@ -53,7 +53,7 @@
     <preference name="DisallowOverscroll" value="false" />
     <preference name="EnableViewportScale" value="false" />
     <preference name="KeyboardDisplayRequiresUserAction" value="true" />
-    <preference name="MediaPlaybackRequiresUserAction" value="false" />
+    <preference name="MediaTypesRequiringUserActionForPlayback" value="false" />
```

Comment:
@erisu FYI, `value="false"` will not work the way you think it will 🙃 

Essentially, `cordovaBoolSettingForKey` expects the value to be a 0 or 1, and turns anything other non-zero value truthy. I kinda consider this to be a bug in `cordovaBoolSettingForKey` tbh.

Reply:
Thank you for bringing this up.

I will take a look at how to improve and fix this behavior so it works the way we expect.

Since my changes are just updating the preference name, for the most part, and that the behavior you are bringing up is applied to all boolean preferences, I will handle it in a separate PR.